sources/platform/actors/publishing/monetize.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Monetize your actor
```

Comment:
@mtrunkat we already have a section on this in Academy (https://docs.apify.com/academy/get-most-of-actors/monetizing-your-actor). Should we just copy paste the content here?

Reply:
Ill put there just minimalistic content and then discuss with Ondra and Percy if we should transition it